canvas_sdk/commands/tests/tests.py

```diff
@@ -456,3 +464,125 @@ def test_command_schema_matches_command_api(
         assert len(expected_field["choices"]) == len(choices)
         for choice in choices:
             assert choice["value"] in expected_field["choices"]
+
+
+@pytest.fixture
+def commands_to_test() -> list[Any]:
+    return [
+        AssessCommand,
+        DiagnoseCommand,
+        GoalCommand,
+        HistoryOfPresentIllnessCommand,
+        MedicationStatementCommand,
+        PlanCommand,
+        PrescribeCommand,
+        QuestionnaireCommand,
+        ReasonForVisitCommand,
+        StopMedicationCommand,
+        UpdateGoalCommand,
+    ]
+
+
+@pytest.fixture
+def new_note_id_and_commands_protocol_code(
+    new_note: dict, commands_to_test: list[Any]
```

Comment:
In addition to test parametrization, you can also parametrize fixtures. Tests that use parametrized fixtures are automatically parametrized themselves. This use case looks like it might fit well.